lib/moon/components/select/single_select/styled/select_value.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Moon.Components.Select.SingleSelect.SelectValue do
+  @moduledoc false
+
+  use Moon.StatelessComponent
+  alias Moon.Components.Select.SingleSelect.Dropdown.Icon
+
+  prop select_id, :string
+  prop option, :any
+  prop label, :string, default: ""
+  prop placeholder, :string, default: ""
+  prop size, :string
+  prop target, :string
+
+  def render(assigns) do
+    is_icon = not is_nil(assigns.option[:left_icon])
+    is_right_icon = not is_nil(assigns.option[:right_icon])
```

Comment:
What is is_right_icon? has_right_icon?

Naming here is hard to read.

Just option.right_icon would work also, why it is required to transform this to is_right_icon?